src/components/BaseSvg.tsx

```diff
@@ -0,0 +1,18 @@
+export default function BaseSvg() {
```

Comment:
let's start introducing a `svg` folder. It will end up useful as we use this template for future more complex templates.